operator/main.go

```diff
@@ -52,6 +53,26 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+// getMetricsBindAddress returns the metrics bind address for the operator
+func getMetricsBindAddress() (string) {
+        var metricsBindAddrEnvVar = "METRICS_BIND_ADDRESS"
+	
+	defaultBindAddr := ":8383"
+
+        bindAddr, found := os.LookupEnv(metricsBindAddrEnvVar)
+        if found {
+		_, err := strconv.Atoi(bindAddr)
+		if err != nil {
+			msg := fmt.Errorf("Supplied METRICS_BIND_ADDRESS is not a number", "METRICS_BIND_ADDRESS: %s", bindAddr)
```

Comment:
This is not correctly formatted hence throwing error while executing `make` command.